sdk/storage/src/blob/blob/mod.rs

```diff
@@ -66,6 +66,52 @@ create_enum!(RehydratePriority, (High, "High"), (Standard, "Standard"));
 
 create_enum!(PageWriteType, (Update, "update"), (Clear, "clear"));
 
+pub mod cr64_optional {
+    use serde::{self, Deserialize, Deserializer};
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<Option<crate::ConsistencyCRC64>, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s: Option<String> = Option::deserialize(deserializer)?;
+        trace!("s == {:?}", s);
+
+        Ok(match s {
+            None => None,
+            Some(s) => match s.len() {
+                0 => None,
+                _ => Some(
+                    crate::ConsistencyCRC64::decode(s)
+                        .map_err(|e| serde::de::Error::custom(e.to_string()))?,
+                ),
+            },
+        })
+    }
+}
+
+pub mod md5_optional {
+    use serde::{self, Deserialize, Deserializer};
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<Option<crate::ConsistencyMD5>, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s: Option<String> = Option::deserialize(deserializer)?;
+        trace!("s == {:?}", s);
```

Comment:
I'd like to not have these trace calls. If you really think it helps, then something a bit more descriptive than `s` would be good.

Reply:
No, you are right, it does have sense to leave it there. Removed!